src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, partnerName } = person;
+  const partnerLabel = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (<p className="Person__age">{`I am ${age}`}</p>)}
+      <p className="Person__partner">{partnerName ? `${partnerName} is my ${partnerLabel}` : `I am not married`}</p>
```

Comment:
```suggestion
      </h2>
      
      {age && (<p className="Person__age">{`I am ${age}`}</p>)}
      
      <p className="Person__partner">
        {isMarried
          ? `${partnerName} is my ${partnerLabel}` 
          : `I am not married`}
      </p>
```
Fix formatting. Also, you should check `isMarried` property of person, not partnerName
![image](https://user-images.githubusercontent.com/96409563/223059880-1dfe6250-8e8b-4c6b-94cf-335129bf345a.png)